Tests/Unit/EmogrifierTest.php

```diff
@@ -1763,4 +1763,27 @@ public function emptyMediaQueriesAreRemoved()
             $result
         );
     }
+
+    /**
+     * @test
+     */
+    public function multiLineMediaQuery()
```

Comment:
Please let's make all test names full sentences that describe some behavior. I propose the name "multiLineMediaQueryIsApplied".